fe/fe-core/src/main/java/com/starrocks/qe/StmtExecutor.java

```diff
@@ -439,6 +447,18 @@ public void execute() throws Exception {
                             parsedStmt = selectStmt;
                             execPlan = StatementPlanner.plan(parsedStmt, context);
                         }
+                    } else if (parsedStmt instanceof ExecuteStmt) {
+                        ExecuteStmt executeStmt = (ExecuteStmt) parsedStmt;
+                        com.starrocks.sql.analyzer.Analyzer.analyze(executeStmt, context);
+                        prepareStmtContext = context.getPreparedStmt(executeStmt.getStmtName());
+                        if (null == prepareStmtContext) {
+                            throw new StarRocksPlannerException(ErrorType.INTERNAL_ERROR,
+                                    "prepare statement can't be found @ %s, maybe has expired", executeStmt.getStmtName());
+                        }
+                        PrepareStmt prepareStmt = prepareStmtContext.getStmt();
+                        parsedStmt = prepareStmt.assignValues(executeStmt.getParamsExpr());
```

Comment:
a question, how to promise the type of value keep same with parameter,  like parameter required `INT`, but the value expression's result may be `BIGINT`

Reply:
and if doesn't analyze after assignValues, may be will use some illegal values?

Reply:
> 
value is literal ,  don't have specific  type,  only use (cast as int). int is target type.  behavior is same like that k is bool column, where k = 3, will get empty result